src/pybind/mgr/mgr_module.py

```diff
@@ -593,6 +595,7 @@ class Option(Dict):
     def __init__(
             self,
             name: str,
+            level: str = "advanced",
```

Comment:
```suggestion
            level: Literal["basic", "advanced", "dev"] = "advanced",
```

Reply:
BTW, given the `level_t` already sets specific values for each enum (0, 1, 2...), why not leveraging that in Python side and simply parsing an `int` instead of a `str`?:
```py
class LevelEnum(enum.IntEnum):
    BASIC = 0
    ADVANCED = 1
    DEV = 2
    UNKNOWN = 3
```
```suggestion
            level: LevelEnum = LevelEnum.ADVANCED,
```

Reply:
fixed, thanks

Reply:
I decided to go a different direction using a `StrEnum`. I'd prefer not to leak the the integer representation from C++ into the python world.

Reply:
`StrEnum` was previously achieved with multiple inheritance:
```py
class LevelEnum(str, enum.Enum): # <-- this exact order
  BASIC = "basic"
  ...
```